BLSQ/iaso
# IA-3713: fix perm validation for geo user w/o org unit

Geo-limited user without org unit should have access to the whole pyramid

Related JIRA tickets : IA-3713

## Self proofreading checklist


## Doc

N/A

## Changes

Explain the changes that were made.

- Only filter by org unit in validation if user profile has org units

## How to test

- Create a geo-limited user with no location
- Login as this user
- Create another geo-limited, but with location
- The user should be created without issue

## Print screen / video

Uploading Screen Recording 2024-11-25 at 11.28.28.mov…


## Notes

Tagged the fix as v1.248a


## Follow the Conventional Commits specification

The **merge message** of a pull request must follow the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) specification.

This convention helps to automatically generate release notes.

Use lowercase for consistency.

[Example](https://github.com/BLSQ/iaso/commit/8b8d7d3064138c1e57878f17b4eb922516ab0112):

```
fix: empty instance pop up

Refs: IA-3665
```

Note that the Jira reference is preceded by a _line break_.

Both the line break and the Jira reference are entered in the _Add an optional extended description…_ field.
